website/demos/map_demos.py

```diff
@@ -13,11 +14,16 @@
 port = 7860
 
 demos_to_run = []
-GRADIO_COMPONENTS_FILE = os.path.join(GRADIO_DIR, "gradio", "components.py")
-with open(GRADIO_COMPONENTS_FILE) as comp_file:
-    comp_text = comp_file.read()
-for demostr in re.findall(r'Demos:(.*)', comp_text):
-  demos_to_run += re.findall(r'([a-zA-Z0-9_]+)', demostr)
+for root, _, files in os.walk(pathlib.Path(GRADIO_DIR) / "gradio"):
+    for filename in files:
```

Comment:
Was able to confirm the demos still build correctly on a local docker compose up

![image](https://user-images.githubusercontent.com/41651716/178549347-90366cef-f2e8-4c30-9da3-06a0d3312200.png)